### PR TITLE
Add an Add Ons button to start screen that loads mp installer

### LINKED ISSUE #1366: Add modpack installer to Start Screen
**Is your feature request related to a problem? Please describe.**
I am not really sure how many people know about the modpack installer as a way to download added content from our third party set of developers. I think it would be best to promote this more.

**Describe the solution you'd like**
Add a "Add Ons" or "Mods" or some other such button to the main start screen. This button will bring up the modpack installer allowing the user to pick what they want.

**Describe alternatives you've considered**
None really. Obviously leaving it alone or putting some text there is another option, but button seems the best solution.

**Additional context**
I was reviewing our `docs` repo and thought of it while reading the page about using the modpack installer.

### PR DESCRIPTION
**Pull request by jwrober**
Closes #1366

Part of #2136

### CODE REVIEW

File: client/fc_client.cpp [hunk 1]
```diff
@@ -687,6 +689,44 @@ void fc_client::start_new_game()
...
+    location = QStandardPaths::findExecutable(modpack_name);
+  }
+
+  // Start it
+  qInfo(_("Starting freeciv21-modpack-qt at %s"), qUtf8Printable(location));
+
+  QProcess *modProcess = new QProcess(this);
+  QStringList arguments;
+  modProcess->start(location, arguments);
+  modProcess->waitForFinished();
```
Comment (lmoureaux) [inline client/fc_client.cpp:727]: We probably don't want to freeze the main window until the installer is closed, potentially triggering dead app dialogs from the system

**Review by lmoureaux — CHANGES_REQUESTED** [review]
On Windows the modpack installer is optional. Should we make it mandatory?

**Comment by jwrober** [comment]
> On Windows the modpack installer is optional. Should we make it mandatory?

Just noticed this. Yes I think it has to be now. I'll push another change.

**Comment by jwrober** [comment]
Turns out the GUI modpack, isn't optional - https://github.com/longturn/freeciv21/blob/master/tools/fcmp/CMakeLists.txt#L34 - the CLI version is so we are good.

**Review by lmoureaux — APPROVED** [review]
We may want to rebrand the installer item as "developer tools" or so then

**Pull request merged**